### DISCUSSION TITLE
Hyperlinks to daily notes in a weekly note

### DISCUSSION THREAD

**Discussion by Hypermensch** [SilentVoid13/Templater#607] [Help]
Is there a script that creates hyperlinks in a weekly note to the daily notes that are included in this weekly note? For example, the 13th week includes these days: 28, 29, 30, 31, 1, 2, 3.

I found a similar script, but it needs to change the days every week. Is it possible to make it so that it does not have to be changed?
1. [[<% tp.date.weekday("YYYY-MM-DD", 0) %>]]
2. [[<% tp.date.weekday("YYYY-MM-DD", 1) %>]]
3. [[<% tp.date.weekday("YYYY-MM-DD", 2) %>]]
4. [[<% tp.date.weekday("YYYY-MM-DD", 3) %>]]
5. [[<% tp.date.weekday("YYYY-MM-DD", 4) %>]]
6. [[<% tp.date.weekday("YYYY-MM-DD", 5) %>]]
7. [[<% tp.date.weekday("YYYY-MM-DD", 6) %>]]

**Comment by liamcain** [comment]
Don't know the answer using Templater, but this should work even without templater installed:

```
## Week at a Glance

![[{{sunday:gggg-MM-DD}}]]
![[{{monday:gggg-MM-DD}}]]
![[{{tuesday:gggg-MM-DD}}]]
![[{{wednesday:gggg-MM-DD}}]]
![[{{thursday:gggg-MM-DD}}]]
![[{{friday:gggg-MM-DD}}]]
![[{{saturday:gggg-MM-DD}}]]
```

**Reply by Hypermensch** [reply]
This script doesn't work...

**Reply by liamcain** [reply]
How are you creating your weekly notes? Are you not using the Periodic Notes plugin?

**Reply by Hypermensch** [reply]
I use this plugin

**Comment by JeromeZephyr** [comment]
My weekly note always has the name this way: YYYY-[W]WW, so 2022-W20 for today.
My dailies have this format: YYYY-MM-DD

The days are in french, you can change them. 

```
| [[<% tp.date.now("YYYY-MM-DD",0-7,tp.file.title,"GGGG-[W]WW") %> \|Lundi]] | [[<% tp.date.now("YYYY-MM-DD",1-7,tp.file.title,"GGGG-[W]WW") %> \|Mardi]] | [[<% tp.date.now("YYYY-MM-DD",2-7,tp.file.title,"GGGG-[W]WW") %> \|Mercredi]] | [[<% tp.date.now("YYYY-MM-DD",3-7,tp.file.title,"GGGG-[W]WW") %> \|Jeudi]] | [[<% tp.date.now("YYYY-MM-DD",4-7,tp.file.title,"GGGG-[W]WW") %> \|Vendredi]] |
|:--- |:--- |:--- |:--- |:--- |


| *[[<% tp.date.now("YYYY-MM-DD",5-7,tp.file.title,"GGGG-[W]WW") %> \|Samedi]]* | *[[<% tp.date.now("YYYY-MM-DD",6-7,tp.file.title,"GGGG-[W]WW") %> \|Dimanche]]* |
|:--- |:--- |

```
